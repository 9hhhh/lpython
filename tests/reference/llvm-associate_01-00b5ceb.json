{
    "basename": "llvm-associate_01-00b5ceb",
    "cmd": "lfortran --show-llvm {infile} -o {outfile}",
    "infile": "tests/../integration_tests/associate_01.f90",
    "infile_hash": "08d039a665a042f2a1d20a71677e2a484073202ed7421a4eb7cf011f",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "llvm-associate_01-00b5ceb.stdout",
    "stdout_hash": "a40a13cb7ca02f42bf9f06391dff87d41a80aab4d4e81c0a0222a695",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}