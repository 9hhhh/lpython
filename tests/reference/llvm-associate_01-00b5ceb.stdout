; ModuleID = 'LFortran'
source_filename = "LFortran"

@0 = private unnamed_addr constant [2 x i8] c"\0A\00", align 1
@1 = private unnamed_addr constant [2 x i8] c"\0A\00", align 1
@2 = private unnamed_addr constant [2 x i8] c"\0A\00", align 1

define i64 @main() {
.entry:
  %a = alloca float, align 4
  %b = alloca float, align 4
  %c = alloca float, align 4
  %n = alloca i64, align 8
  %x = alloca float, align 4
  %y = alloca float, align 4
  store i64 10, i64* %n, align 4
  call void (i8*, ...) @_lfortran_printf(i8* getelementptr inbounds ([2 x i8], [2 x i8]* @0, i32 0, i32 0))
  call void (i8*, ...) @_lfortran_printf(i8* getelementptr inbounds ([2 x i8], [2 x i8]* @1, i32 0, i32 0))
  call void (i8*, ...) @_lfortran_printf(i8* getelementptr inbounds ([2 x i8], [2 x i8]* @2, i32 0, i32 0))
  ret i64 0
}

declare void @_lfortran_printf(i8*, ...)

